modules/module-info.php

```diff
@@ -898,13 +898,30 @@ function jetpack_custom_jetpack_manage() { ?>
 
 // XML Sitemap: START
 function jetpack_xml_sitemap_more_info() { ?>
-	<p><?php esc_html_e( 'This module creates an XML sitemap file that lists the URLs of posts and pages in your site with important information about each one.', 'jetpack' ); ?></p>
-	<p><?php esc_html_e( 'This file is accessed by search engines like Google or Bing so they can crawl and understand your site.', 'jetpack' ); ?></p>
-	<p>&rarr; <a href="http://jetpack.me/support/sitemaps/"><?php esc_html_e( 'More information on Sitemaps.', 'jetpack' ); ?></a></p>
+	<p><?php esc_html_e( 'This module creates two XML sitemap files that list the URLs of posts and pages in your site with important information about each one. This files can be accessed by search engines like Google or Bing so they can crawl and understand your site.', 'jetpack' ); ?></p>
 	<?php if ( '0' == get_option( 'blog_public' ) ) : ?>
 		<p><strong><?php esc_html_e( 'Your site is currently set to discourage search engines from indexing it so the sitemap will not be accessible.', 'jetpack' ); ?></strong></p>
+	<?php else :
+		$sitemap_url = home_url( get_option( 'permalink_structure' ) ? '/sitemap.xml' : '/?jetpack-sitemap=true' );
+		$news_sitemap_url = home_url( get_option( 'permalink_structure' ) ? '/news-sitemap.xml' : '/?jetpack-news-sitemap=true' );
```

Comment:
Because it mimics the settings from the Sitemap module:
https://github.com/Automattic/jetpack/blob/master/modules/sitemaps/sitemaps.php#L618
Feels kinda anti-DRY if you ask me :)